docs/da/index.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Den officielle dokumentation for AtlasOS
+icon: material/home
+---
+
+# Velkommen til Atlas' Dokumentation
+
+<div class="grid cards" markdown>
+
+-   :material-power:{ .lg .middle } __Installer AtlasOS__
+
+    ---
+
+    start op med at få AtlasOS sat op på dit system på blot et par minutter.
+
+    [:octicons-arrow-right-24: Install AtlasOS](getting-started/installation.md)
```

Comment:
The link name here, "Install AtlasOS", can also be translated if there is a translation available in Danish. This applies to the 3 other links further down in the file, too. :)

Reply:
Oh, didn't see that lol